src/dmd/parse.d

```diff
@@ -6492,6 +6493,18 @@ final class Parser(AST) : Lexer
             {
                 switch (t.value)
                 {
+                case TOK.leftParentheses:
+                    parens++;
+                    continue;
+                case TOK.rightParentheses:
+                    parens--;
+                    continue;
+                // https://issues.dlang.org/show_bug.cgi?id=21163
+                // lambda param can has the `scope` STC
+                // `S s = { (scope Type Id){} }`
```

Comment:
"can have"

I don't think the bug needs to be referenced here

What is STC?  Just say "keyword"?

Reply:
- have: ok.
- bugs are referenced everyhwere in the code comments.
- STC is commnly used in the DMD codebase for "Storage Class".

Reply:
> bugs are referenced everyhwere in the code comments.

Yes, and it can be really useful when you're looking at some code and trying to understand the motivation for a special can be hard as with older code bases the git history/blame often don't work well.

Reply:
But here a syntax example is given in the comment, and that is sufficient.  The bug link is extraneous.

Reply:
The "leave a bug link for every change" does not scale. Given enough time, the source code will be littered with these comments and so less readable.  A skilled git use can follow the trail in the rare case it's necessary.  So I strongly disagree.

Reply:
I have no strong opinion on this. The fact that I added the link is more a sign of assimilation to the most common practices I'd say. But I can remove it as I have to force push a 2nd time due to DAutoText failing for no reasons.